src/libs/actions/Welcome/index.ts

```diff
@@ -173,19 +174,15 @@ function completeHybridAppOnboarding() {
         },
     ];
 
-    const failureData: OnyxUpdate[] = [
-        {
-            onyxMethod: Onyx.METHOD.MERGE,
-            key: ONYXKEYS.NVP_TRYNEWDOT,
-            value: {
-                classicRedirect: {
-                    completedHybridAppOnboarding: false,
-                },
-            },
-        },
-    ];
+    // eslint-disable-next-line rulesdir/no-api-side-effects-method
+    API.makeRequestWithSideEffects(SIDE_EFFECT_REQUEST_COMMANDS.COMPLETE_HYBRID_APP_ONBOARDING, {}, {successData}).then((response) => {
+        if (!response) {
+            return;
+        }
 
-    API.write(WRITE_COMMANDS.COMPLETE_HYBRID_APP_ONBOARDING, {}, {optimisticData, failureData});
+        Log.info(`[HybridApp] Onboarding status has changed. Propagating new value to OldDot`, true, {completedHybridAppOnboarding: response?.jsonCode === 200});
```

Comment:
If the onboarding complete status is always going to be set to true, can we make that a bit clearer in the log message. To help people unfamiliar with HybridApp